docs/scripts/formattedTSDemos.js

```diff
@@ -87,22 +100,25 @@ async function transpileFile(filePath) {
 
   let successful = 0;
   let failed = 0;
-  (await Promise.all(tsxFiles.map(transpileFile))).forEach(item => {
-    if (item) successful += 1;
-    else failed += 1;
+  let skipped = 0;
+  (await Promise.all(tsxFiles.map(transpileFile))).forEach(result => {
+    if (result === 0) successful += 1;
```

Comment:
Rather use some descriptive const enum value here. It is apparent in the immediate context but the context is lost in the return statement of `transpileFile`.

Reply:
Done